config/charts/inferencepool/templates/epp-deployment.yaml

```diff
@@ -27,6 +27,45 @@ spec:
       # Conservatively, this timeout should mirror the longest grace period of the pods within the pool
       terminationGracePeriodSeconds: 130
       containers:
+      {{- if .Values.inferenceExtension.sidecar.enabled }}
+      - name: sidecar
+        image: {{ .Values.inferenceExtension.sidecar.image.hub }}/{{ .Values.inferenceExtension.sidecar.image.name }}:{{ .Values.inferenceExtension.sidecar.image.tag }}
```

Comment:
`{{ .Values.inferenceExtension.sidecar.image }}` is sufficient, no need to allow setting the various parts explicitly